src/java.base/share/classes/sun/security/util/KeyUtil.java

```diff
@@ -174,6 +174,22 @@ public static final int getKeySize(AlgorithmParameters parameters) {
         return -1;
     }
 
+    /**
+     * If the key is a sub-algorithm of a larger group of algorithms, this
+     * method will return that sub-algorithm.  For example, key.getAlgorithm()
+     * returns "EdDSA", but the underlying key may be "Ed448".  For
+     * DisabledAlgorithmConstraints (DAC), this distinction is important.
+     * "EdDSA" means all curves for DAC, but when using it with
+     * KeyPairGenerator, "EdDSA" means "Ed25519".
```

Comment:
Maybe just say this allows more precise check for DAC. For `KeyPairGenerator`, "EdDSA" by default means "Ed25519", but you can always call `init(NamedParameterSpec.ED448)` to change it.

Reply:
I see what your saying, but I was only explaining when EdDSA & Ed25519 can mean the same with KPG.  As this is an internal method, I wasn't trying to explaining how to generate an Ed488 key.